### ISSUE TITLE
Documentation should tell to load the TelemetryDeck SDK async?

### ISSUE THREAD

**Issue by Timac** [TelemetryDeck/WebSDK#9]
The installation instructions of TelemetryDeck for a website are:

```
<script
  src="https://cdn.telemetrydeck.com/websdk/telemetrydeck.min.js"
  data-app-id="<YOUR APP ID>"
></script>
```

This means that downloading the TelemetryDeck SDK is blocking the browser. Shouldn't `async` be used to tell the browser to download the script in parallel with the rest of the page? Or am I missing something because I don't think that TelemetryDeck need to block the rendering of your page?

```
<script async
  src="https://cdn.telemetrydeck.com/websdk/telemetrydeck.min.js"
  data-app-id="<YOUR APP ID>"
></script>
```

See the documentation here:
- [https://github.com/TelemetryDeck/WebSDK](https://github.com/TelemetryDeck/WebSDK)
- [https://telemetrydeck.com/docs/guides/web-setup/](https://telemetrydeck.com/docs/guides/web-setup/)

**Comment by pichfl** [comment]
Yes, we should add the keyword. No, there is no need for the script to be blocking.
